packages/core/src/mesh/ModelMesh.ts

```diff
@@ -14,4 +66,793 @@ export class ModelMesh extends Mesh {
     super(engine);
     this.name = name;
   }
+
+  /**
+   * Upload Mesh Data to the graphics API.
+   * @param noLongerAccessible - Whether to access data later. If true, you'll never access data anymore (free memory cache)
+   */
+  uploadMeshData(noLongerAccessible: boolean): void {
+    const { _indicesArray, _vertexSlotChanged, _accessible } = this;
+    if (!_accessible) {
+      throw "Not allowed to access data while accessible is false.";
+    }
+
+    // Structure of vertex buffer has changed
+    if (_vertexSlotChanged || this._vertexCountChanged) {
+      const vertexElements = this._updateVertexElements();
+      const elementCount = this._elementCount;
+      const vertices = new Float32Array(elementCount * this.vertexCount);
+      this._verticesArray = vertices;
+      this._vertexChangeFlag = ValueChanged.All;
+      this._updateVertices(vertices);
+      this._vertexBuffer = new Buffer(
+        this._engine,
+        BufferBindFlag.VertexBuffer,
+        vertices,
+        noLongerAccessible ? BufferUsage.Static : BufferUsage.Dynamic
+      );
+      this.setVertexElements(vertexElements);
+      this.setVertexBufferBinding(new VertexBufferBinding(this._vertexBuffer, elementCount * 4));
```

Comment:
You can write "this.setVertexBufferBinding(this._vertexBuffer, elementCount * 4);" to simplify code.